### ISSUE TITLE
Some plans not working after restoring backup

### ISSUE THREAD

**Issue by Popkornium18** [mtotschnig/MyExpenses#1149]
I successfully restored a backup to another phone and all the plans were also imported successfully. I can see them in the Plans and Templates screen of the app. However I can only find some of them in the calender that MyExpenses uses. Only these plans are actually executed. The other plans are not. All of these plans worked fine on the other phone.

Also: Is there a way to manually trigger a plan? I'd like to avoid manually adding expenses that match the plan.

**Comment by mtotschnig** [comment]
Are the plans that are not visible in the calendar still shown as plans (with the calendar icon: ![grafik](https://user-images.githubusercontent.com/1485456/223516785-dbc18d9b-68e1-47c2-9ed4-cb227ff4034f.png))
or as template (![grafik](https://user-images.githubusercontent.com/1485456/223516978-67695fe3-a8bd-439b-aed8-13022a7095bc.png)) ?

**Comment by Popkornium18** [comment]
They are shown as plans

**Comment by mtotschnig** [comment]
When you open the calendar dialog for the plan, does it highlight the instances for the plan correctly?

**Comment by Popkornium18** [comment]
No, when I open the dialog for one of the broken plans nothing is highlighted.

**Comment by Popkornium18** [comment]
In case anyone else is affected: As a workaround you can fix your plans again by changing the date and then changing it back.

**Comment by mtotschnig** [comment]
Sorry, forgot to answer this question:

> Is there a way to manually trigger a plan?
On the calendar view, clicking on the highlighted cells that indicate the instances of the plan, gives you the option to trigger the plan for that instance.

But in your case it seems that the information about instances was not available, since you reported that they did not appear on the calendar. 
I do not have an explanation yet, how this can happen only for some plans, and why changing the back would fix it. I will to try to reproduce the problem.
